dotenv-linter/src/cli.rs

```diff
@@ -130,6 +126,11 @@ fn common_args() -> Vec<Arg> {
             .long("recursive")
             .help("Recursively searches and checks .env files")
             .action(ArgAction::SetTrue),
+        Arg::new("schema")
+            .short('S')
+            .long("schema")
+            .help("Use schema file to check .env files")
+            .value_parser(value_parser!(PathBuf)),
```

Comment:
Btw, it can be useful to have an ability to pass schema as an in-place argument (json string) to the CLI.

E.g it can be used when you're piping commands